### PR TITLE
Update manifests.md

### PR DESCRIPTION
**Pull request by clemenko**

### CODE REVIEW

File: docs/guides-references/manifests.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ metadata:
   annotations:
     # global flags for all images in the manifest
     # image flags override global flags
+    # example, hauler.dev/registry: will only prepend if the server reference is missing
```
Comment (zackbradys) [inline docs/guides-references/manifests.md:19]:
```suggestion
    # example: platform set globally, but not observed if set per image
    # example: registry set globally, but not observed if set per image
```

Comment (zackbradys) [reply docs/guides-references/manifests.md:19]: Please add changes to `docs/guides-references/hauler-content/images.md` as well.

**Review by zackbradys — CHANGES_REQUESTED** [review]
Please add the changes to the two other occurrences of these comments in `docs/guides-references/hauler-content/images.md`.

**Comment by clemenko** [comment]
completed.

**Review by zackbradys — APPROVED** [review]
LGTM! Thanks for the updates

**Pull request merged**